packages/sdk-snippets/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
+    "module": "nodenext",
+    "moduleResolution": "nodenext",
```

Comment:
can we remove these lines? i believe this can be inferred from the `tsconfig.build.json`

```suggestion
```